test/cli_ps_test.go

```diff
@@ -107,6 +107,21 @@ func (suite *PouchPsSuite) TestPsQuiet(c *check.C) {
 	}
 }
 
+// TestPsNoTrunc tests "pouch ps trunc" work
+func (suite *PouchPsSuite) TestPsNoTrunc(c *check.C) {
+	name := "ps-noTrunc"
+
+	command.PouchRun("create", "--name", name, busyboxImage).Assert(c, icmd.Success)
+	defer DelContainerForceMultyTime(c, name)
+
+	command.PouchRun("start", name).Assert(c, icmd.Success)
+
+	res := command.PouchRun("ps", "--no-trunc").Assert(c, icmd.Success)
+	kv := psToKV(res.Combined())
+
+	c.Assert(kv[name].id, check.HasLen, 64)
```

Comment:
The check maybe too simple:
we may execute `pouch inspect CONTAINERNAME`  to get the ContainerID, then execute `pouch ps -q --no-trunc` to get the the ContainerID again, then we check whether the IDs are equaled.  WDYT?

Reply:
Right. Your opinion is more thoughtful. :)